src/java/com/puppetlabs/http/client/impl/JavaClient.java

```diff
@@ -57,8 +58,7 @@
 import java.security.NoSuchAlgorithmException;
 import java.security.cert.CertificateException;
 import java.security.cert.X509Certificate;
-import java.util.HashMap;
-import java.util.Map;
+import java.util.*;
```

Comment:
Yeah, I think I had to do something like this:

Editor -> Code Style -> Java -> Imports ->
  Class count to use import with '*' -> 9999
  Names count to use static import with '*': 9999

Not great but seems to work.